apps/charterafrica/src/pages/index.page.js

```diff
@@ -307,6 +311,21 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
             },
           ],
         },
+        {
+          slug: "mooc",
+          title: {
+            content:
+              "Learn how to use digital tools to <i>strengthen</i> your democracy 1",
+            color: "common.white",
+          },
+          link: {
+            content: "Learn more",
+            color: "primary",
+          },
+          image: {
+            src: "/images/mooc.png",
+          },
```

Comment:
Use something similar to what we'll get from CMS.
```suggestion
          title: [
            children: [
              {
                text: "Learn how to use digital tools to <i>strengthen</i> your democracy 1",
                children: null,
              },
            ],
          ],
          link: {
            label: "Learn more",
          },
          image: {
            url: "http://localhost/images/mooc.png",
          },
```

Don't think we need the visual customisations.